src/precompile.jl

```diff
@@ -0,0 +1,30 @@
+using PrecompileTools: @setup_workload, @compile_workload
+
+@setup_workload begin
+    # These need to be safe to call here and bake into the pkgimage, i.e. called twice.
+    Connections.__init__()
```

Comment:
I don't understand why these are needed. The `__init__` of dependencies should run during precompilation.

Reply:
They're modules not dependencies.

Reply:
Oh, duh...